lib/code_fund_web/slack.ex

```diff
@@ -0,0 +1,45 @@
+defmodule CodeFundWeb.SlackBot do
+  require Logger
+  use Slack
+
+  @doc """
+  Sends message to channel use Web api.
+  """
+  @spec web_send_message(String.t()) :: any
+  def web_send_message(message) do
+    token = Application.get_env(:code_fund, CodeFundWeb.Endpoint)[:slack_token]
+
+    if token do
+      channel = Application.get_env(:code_fund, CodeFundWeb.Endpoint)[:slack_channel]
+      avatar = Application.get_env(:code_fund, CodeFundWeb.Endpoint)[:slack_avatar]
+      bot_name = Application.get_env(:code_fund, CodeFundWeb.Endpoint)[:slack_bot_name]
+
+      Slack.Web.Chat.post_message(channel, message, %{
+        icon_url: avatar,
+        username: bot_name
+      })
+    end
+  end
+
+  def handle_connect(slack, state) do
+    Logger.debug(fn -> "Connected as #{slack.me.name}" end)
+    {:ok, state}
+  end
+
+  def handle_event(%{type: "message"} = message, slack, state) do
```

Comment:
`slack` variable unused, please underscore this variable if required for the function.

Reply:
ok